QuickOSM/core/query_preparation.py

```diff
@@ -202,7 +203,8 @@ def replace(catch, default_nominatim):
 
             if lat is None and lon is None:
                 if not self._nominatim:
-                    self._nominatim = Nominatim()
+                    server = get_setting('defaultNAPI', NOMINATIM_SERVERS[0])
```

Comment:
`defaultNAPI` -> `defaultNominatimAPI`, it's way more readable.
It was a bad choice of using `OAPI`.